### DISCUSSION TITLE
About ltsp.conf - IP globs

### DISCUSSION THREAD

**Discussion by AsmoKoskinen** [ltsp/ltsp#163] [Q&A]
There is an example in ltsp.conf file.

```
# Globs can be used in MAC, IP, or hostname sections.
# This matches all Raspberry PIs.
[b8:27:eb:*]
# HOSTNAME_PREFIX=rpi
```

Raspberry Pi 4 comes with new OUI.
[dc:a6:32:*]

So this new OUI should also be added in example if keeping Raspberry Pi example.

See issue ltsp/community#76 with PiServer:

https://github.com/raspberrypi/piserver/issues/76

**Comment by alkisg** [comment]
Thank you Asmo, it's not very significant to put it in the example, but it's very significant to put it in ltsp-dnsmasq.conf, so that pi4 clients will be able to netboot.

Will commit it tomorrow.

**Comment by alkisg** [comment]
Fix committed in https://github.com/ltsp/ltsp/commit/b13919e6a42ce2f1aeae22cffb17520b3655500e.

**Comment by alkisg** [comment]
Hmm, I wonder if this section works:

```shell
[b8:27:eb:*|dc:a6:32:*]
HOSTNAME_PREFIX=rpi
```

As the implementation is using shell's "case" command, it might actually work...

**Reply by intelliant01** [reply]
Works. :)

**Reply by intelliant01** [reply]
Precisely, the section works but `HOSTNAME_PREFIX="rpi"` doesn't.

**Comment by alkisg** [comment]
HOSTNAME_PREFIX has been removed 2 years ago: https://github.com/ltsp/ltsp/issues/57#issuecomment-548705766
